README.md

```diff
@@ -50,162 +43,112 @@ source and public models in popular formats such as TensorFlow, ONNX, PaddlePadd
-            <td><b><i><a href="./src/plugins/auto">openvino_auto_plugin</a></i></b></td>
-            <td>Multi plugin enables simultaneous inference of the same model on several devices in parallel</td>
-        </tr>
-    </tbody>
-</table>
-
-## License
-OpenVINO™ Toolkit is licensed under [Apache License Version 2.0](LICENSE).
-By contributing to the project, you agree to the license and copyright terms therein and release your contribution under these terms.
+### Getting Started
 
-## Telemetry
-OpenVINO™ collects software performance and usage data for the purpose of improving OpenVINO™ tools. This data is collected directly by OpenVINO™ or through the use of Google Analytics 4.
-You can opt-out at any time by running the command:
+[OpenVINO Quickstart example](https://docs.openvino.ai/2024/notebooks/201-vision-monodepth-with-output.html) will walk you through the basics to deploy your first model.
 
-``` bash
-opt_in_out --opt_out
-```
+### Installation
+
+Get your OpenVINO installation command with just a few clicks. [Go to the installation page](https://docs.openvino.ai/2024/openvino_docs_install_guides_overview.html).
 
-More Information is available at https://docs.openvino.ai/latest/openvino_docs_telemetry_information.html.
+Check [system requirements](https://docs.openvino.ai/2024/system_requirements.html) and [supported devices](https://docs.openvino.ai/2024/openvino_docs_OV_UG_supported_plugins_Supported_Devices.html) for detailed information.
 
-## Documentation
+### Tutorials
 
-### User documentation
+Explore a variety of tutorials in the [OpenVINO Notebooks Repository](https://github.com/openvinotoolkit/openvino_notebooks)📚.
 
-The latest documentation for OpenVINO™ Toolkit is available [here](https://docs.openvino.ai/). This documentation contains detailed information about all OpenVINO components and provides all the important information you may need to create an application based on binary OpenVINO distribution or own OpenVINO version without source code modification.
+Check out these notebooks that show how to optimize and deploy popular models (no installation required):
+- [Create an LLM-powered Chatbot using OpenVINO](https://github.com/openvinotoolkit/openvino_notebooks/blob/main/notebooks/254-llm-chatbot/254-llm-chatbot.ipynb)
+- [YOLOv8 Optimization](https://github.com/openvinotoolkit/openvino_notebooks/tree/main/notebooks/230-yolov8-optimization)
+- [Text-to-Image Generation](https://github.com/openvinotoolkit/openvino_notebooks/tree/main/notebooks/235-controlnet-stable-diffusion)
 
-### Developer documentation
+### Products Powered by OpenVINO
 
-[Developer documentation](./docs/dev/index.md) contains information about architectural decisions which are applied inside the OpenVINO components. This documentation has all necessary information which could be needed in order to contribute to OpenVINO.
+#### OpenVINO Ecosystem
```

Comment:
Tools in this section are technically "powered by OV", but maybe we could consider to highlight them as part of official "Development toolkit", and move to dedicated section? 
@moslex @andrei-kochin @MaximProshin calling for your opinion here :)

Reply:
If you think that ecosystem word is confusing here, we can state it as OpenVINO workflow projects 
![image](https://github.com/openvinotoolkit/openvino/assets/38211585/762371cd-9350-448f-8226-dbe0c3102861)
We can add openvino_tokenizers as well here in the list https://github.com/openvinotoolkit/openvino_tokenizers.
May be we can even add this picture or link to it.